Rakefile

```diff
@@ -4,6 +4,7 @@ task :test do
   options = {
     allow_hash_href: true,
     assume_extension: true,
+    disable_external: true,
```

Comment:
This disables the test suite from checking external urls like `sss.gov`.  sss.gov particularly is the cause of intermittent test failures.  It can take >30 seconds to respond.

Is this a tradeoff we're willing to make?

Reply:
I think so. We can always run this periodically to check the health of the links?